nbs/03_matcher.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<p>again where are the tests?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>